packages/p2p/src/components/my-ads/edit-ad-form.jsx

```diff
@@ -149,6 +150,7 @@ const EditAdForm = () => {
                             max_transaction: max_order_amount_display,
                             min_transaction: min_order_amount_display,
                             offer_amount: amount_display,
+                            //order_completion_time: order_completion_time,
```

Comment:
could we add a comment above it to say to uncomment this once BE has been released for order_completion_time 👀